File: lib/layer2_core/dark/e_on_error_container_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EOnErrorContainerDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd44;
      case 'standard': return cstd44;
      case 'blue': return cbd44;
      case 'green': return cgd44;
      case 'red': return crd44;
      case 'nothing': return cnod44;
      default: return cstd44;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EOnErrorContainerDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EOnErrorContainerDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd44
            AppTheme.STANDARD -> R.color.cstd44
            AppTheme.BLUE -> R.color.cbd44
            AppTheme.GREEN -> R.color.cgd44
            AppTheme.RED -> R.color.crd44
            AppTheme.NOTHING -> R.color.cnod44
        }
    }
}
